core/metacling/test/TClingTests.cxx

```diff
@@ -147,18 +151,35 @@ TEST_F(TClingTests, GetClassSharedLibs)
    // note the missing space.
 }
 
-static std::string MakeDepLibsPlatformIndependent(llvm::StringRef libs) {
-   llvm::SmallVector<llvm::StringRef, 32> splitLibs;
-   libs.trim().split(splitLibs, ' ');
+static std::string MakeDepLibsPlatformIndependent(const std::string &libs) {
+   auto trim = [](const std::string &s) {
+      std::string ret = s;
+      while (!ret.empty() && std::isspace(ret[0]))
+         ret.erase(0, 1);
+      while (!ret.empty() && std::isspace(ret[ret.size() - 1]))
+         ret.erase(ret.size() - 1, 1);
+      return ret;
+   };
+
+   auto split = [](const std::string &s) -> std::vector<std::string> {
+      std::vector<std::string> ret;
```

Comment:
Out of curiosity why the switch from `llvm::SmallVector<llvm::StringRef, 32>` to ` std::vector<std::string>` ?

Reply:
To remove the dependency on libCling internals; see the corresponding commit log.